lib/active_admin/view_helpers/download_format_links_helper.rb

```diff
@@ -2,6 +2,18 @@ module ActiveAdmin
   module ViewHelpers
     module DownloadFormatLinksHelper
 
+      def build_download_formats(download_links)
+        download_links = instance_exec(&download_links) if download_links.is_a?(Proc)
+
+        if download_links.is_a?(Array) && !download_links.empty?
```

Comment:
@varyonic , since it can be result of custom block, technically it can be everything